R/controls.R

```diff
@@ -486,7 +484,7 @@ animation_options <- function(interval=1000,
 #'   The width vector will be recycled if there are more \code{...} arguments.
 #'   \code{NA} columns will evenly split the remaining columns that are left
 #'   after the widths are recycled and non-\code{NA} values are subtracted.
-#' @param device The class of device which is targeted by these widths; with
+#' @param device Deprecated. The class of device which is targeted by these widths; with
```

Comment:
```suggestion
#' @param device The class of device which is targeted by these widths; with
```

Reply:
My bad, sorry. Relic of when I totally removed bootstrap.